app.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "SVG to Image Proxy",
+  "description": "A web service to create PNGs from SVGs",
+  "keywords": ["svg", "png", "image", "proxy"],
+  "website": "https://shields.io/",
+  "repository": "https://github.com/badges/svg-to-iamge-proxy",
+  "logo": "httpx://shields.io/favicon.png",
```

Comment:
We do not have favicon at https://shields.io/favicon.png, but why httpx://shields.io/favicon.png :-)? It this intentional?

Reply:
Nope, I mistyped!

Reply:
@platan Any interest in taking a crack at getting this running on Heroku?